apis/ops/v1alpha1/singlestore_ops_types.go

```diff
@@ -69,15 +69,17 @@ type SinglestoreOpsRequestSpec struct {
 	Configuration *SinglestoreCustomConfigurationSpec `json:"configuration,omitempty"`
 	// Specifies information necessary for configuring TLS
 	TLS *TLSSpec `json:"tls,omitempty"`
+	// Specifies information necessary for configuring authSecret of the database
+	Auth *AuthSpec `json:"auth,omitempty"`
```

Comment:
May be we should name this field - `authentication` . Shortform auth generally refers to authentication but it also resembles authorization. Besides, other ops fields are in full form.